src/test/components/ReviewCard.test.tsx

```diff
@@ -14,6 +14,16 @@ const mockReview = {
   tags: [{ name: 'Communicative' }, { name: 'Supportive' }],
 } as Review
 
+const mockReviewWithPublishedComment = {
+  ...mockReview,
+  comment: { body: 'This is a test review', status: 'published' },
+} as Review
+
+const mockReviewWithUnpublishedComment = {
+  ...mockReviewWithPublishedComment,
+  comment: { status: 'unpublished' },
```

Comment:
I know it doesn't really affect the test outcome currently, but this should have a 'body' prop to be a proper comment object. Might help us catch errors if the tests are updated down the road.